whipper/common/program.py

```diff
@@ -470,6 +470,47 @@ def getHTOA(self):
         stop = track.getIndex(1).absolute - 1
         return start, stop
 
+    def getCoverArt(self, path, cover_art, release_id):
```

Comment:
The docstring is missing information about the `cover_art` argument (don't know if this name is ideal).

Reply:
> (don't know if this name is ideal).
yeah i think so too the original name was `fetch_cover_art`.

Reply:
and it should be `coverart` instead of `cover_art` tho.

Reply:
Isn't it just a boolean variable, don't know if the name makes it clearer enough

Reply:
`fetch_cover_art` would probably be a better name.

(Also, it should indeed be `cover_art` and not `coverart`, since it’s "cover art" when written normally. Same reason that camelCasing it makes it `getCoverArt` and not `getCoverart`.)